pom.xml

```diff
@@ -19,8 +19,8 @@
   <parent>
     <groupId>com.azure</groupId>
     <artifactId>azure-sdk-parent</artifactId>
-    <version>1.6.0</version> <version>1.6.0</version> <!-- {x-version-update;com.azure:azure-sdk-parent;current} -->
-    <relativePath>./parent/pom.xml</relativePath>
+    <version>1.6.0</version> <!-- {x-version-update;com.azure:azure-sdk-parent;current} -->
+    <relativePath>sdk/parents/azure-sdk-parent</relativePath>
```

Comment:
This is a general question about the relative paths across the pom files for their parent references. Is there any reason why the previous relative path tags included the xml file and the new relative path tags do not? I realize with your restructuring there's only one pom.xml in each of those parent directories but I'm just curious.

Reply:
Maven will by default append ```pom.xml``` when processing the relative path, so its redundant. It was redundant before, but with the restructuring now to have client, management and data poms in separate directories it is more redundant since there is no opportunity for confusion.